org.radare.Cutter.yml

```diff
@@ -0,0 +1,47 @@
+id: org.radare.Cutter
```

Comment:
File needs some adjustments to follow our YAML style guide: https://github.com/flathub/flathub/wiki/YAML-Style-Guide

Reply:
✓